### PR TITLE
Pull Request that adds functionality to scan GenAI apps using Burpsuite exported request file

### PR DESCRIPTION
**Pull request by Ab1z3r**
Hello! I have made some changes to the codebase to add the functionality of request file import.

Trying to follow OOP concepts I created seperate classes and also made a RequestModel class which would make request modification much easier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,20 +107,49 @@ options:
                         [[CHAKRA]] [CHAKRA] CHAKRA <<CHAKRA>>
 ```
 
-### More Examples
+### As GenAI Mobile Application Scanner
 
+It works as follows: 
+1. Use Burpsuite in tandum with Andoird emulator to intercept request made to GenAI application
```
Comment (jchauhan) [inline README.md:113]: You need to mention to specify the marker either while interception or after interception of the request

File: chakra/main.py [hunk 2]
```diff
@@ -59,11 +59,24 @@ def main():
     webapps_parser.add_argument("-l", "--log_level", type=str, default="INFO", help="Log Levels - DEBUG, INFO, WARN, ERROR. Default: INFO")
     webapps_parser.add_argument("--marker", type=str, default="", help=f"FUZZ marker. By Default, the tool will detect any of these markers: {' '.join(FUZZING_MARKERS)}")
 
-
     # Subparser for scanning models
     # models_parser = subparsers.add_parser('models', help='Scan models')
     # models_parser.add_argument("model_path", type=str, help="Path to the model to scan.")
 
+    # Subparser for scanning Burp Resquest files - Mobile Apps
+    webapps_parser = subparsers.add_parser('mobileapp', help='Scan burp request from Mobile App.')
```
Comment (jchauhan) [inline chakra/main.py:67]: The variable name should be mobile_parser

File: chakra/webapp/har.py [hunk 3]
```diff
@@ -6,7 +6,10 @@
 import logging
 from urllib.parse import urlparse, parse_qs, urlencode, urlunparse
 from haralyzer import HarParser, HarPage
-from .model import WebappRemoteModel
+from .model import WebappRemoteModel, RequestModel, MobileAppRemoteModel
```
Comment (jchauhan) [inline chakra/webapp/har.py:9]: You can create another file called burp.py and move your code there. Try to maintain the code separation.

**Review by jchauhan — CHANGES_REQUESTED** [review]
I have provided few comments. Make these changes please.

**Comment by Ab1z3r** [comment]
Just added the changes mentioned above

**Review by jchauhan — APPROVED** [review]

**Pull request merged**